### PR TITLE
[DFFE-11] Sign-Up Form Validation

### PR DESCRIPTION
**Pull request by lorlahdey**
@Ireti203

### CODE REVIEW

File: submissions/lawalLolade/DOM/signUp.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+const signUp = document.getElementById('Form');
```
Comment (Ireti203) [inline submissions/lawalLolade/DOM/signUp.js:1]: This Id is better written as 'form'

**Review by Ireti203 — CHANGES_REQUESTED** [review]
I expect to see where the .js file is linked in your signUp.html file but this PR doesnt show that.

**Comment by lorlahdey** [comment]
@Ireti203

**Review by Ireti203 — APPROVED** [review]

**Review by adaugochi — CHANGES_REQUESTED** [review]
You didn't write a function to check  if password 1 matches password 2

**Comment by lorlahdey** [comment]
@adaugochi @Ireti203

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**